textproc/libunistring/Portfile

```diff
@@ -30,9 +30,6 @@ long_description    Text files are nowadays usually encoded in Unicode, and may
 homepage            https://www.gnu.org/software/libunistring/
 master_sites        gnu
 
-depends_build       bin:perl:perl5 \
-                    port:texinfo
```

Comment:
The commit message should mention texinfo, because that's the main dependency being removed (perl is only a bin: dependency which is satisfied by the system perl), and also explain that the reason it isn't needed is that the release tarball already has the docs built.

Reply:
I'll fix the commit message.

Reply:
Done in 80f2a89286a